examples/org-viewer-react/src/OrgInfo.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useOrganization, Organization } from '@aragon/connect-react'
+import Group from './Group'
+import Table from './Table'
+
+export default function OrgApps() {
+  const [org] = useOrganization()
+
+  if (!org) {
+    return null
+  }
+
+  return (
+    <Group name="Organization">
```

Comment:
A few of the other components uses `loading` with `Group`—just wanted to double check we didn't want to do that here!

Reply:
I refactored the loading states a bit, so that every box is doing the same thing now.